python/pyspark/ml/tests.py

```diff
@@ -371,6 +378,126 @@ def test_fit_maximize_metric(self):
         self.assertEqual(1.0, bestModelMetric, "Best model has R-squared of 1")
 
 
+class RegressorTest(PySparkTestCase):
```

Comment:
@Lewuathe Sorry for late response. What I means is that we can add test for ```numFeatures``` in the current test cases of each algorithm. For example, we can add 
```
>>> model.numFeatures
2
``` 
after [here](https://github.com/apache/spark/blob/master/python/pyspark/ml/classification.py#L55) for ```LogisticRegressionModel```.
The python doctest can also used as examples that users can get how to use ```numFeatures``` from the test case  context.